.github/ISSUE_TEMPLATE.md

```diff
@@ -1 +1,16 @@
-<!-- Before opening a new issue, please check that no existing issue covers the same topic -->
+<!-- Before opening a new issue, please check that no existing issue covers the same topic
+
+This is the template for a feature request.
+For bug reports, refer to: docs/template-bugReport.md
+For new contributor introduction, refer to: docs/template-contributorIntro.md
```

Comment:
Hmmm I don't think we should have a default, because whatever the default is, we have to create two duplicates of it (once here, and once in the original template).
Instead, make good use of `CONTRIBUTING.md`, which is already present by default when anyone's opening issue. You can have something like:
```markdown
Our contributing guidelines (link given above) contain some templates to be used when posting a new issue. We encourage you to use them as appropriate.
```